src/renderer/shared/ui/Buttons/ButtonIcon/common/constants.ts

```diff
@@ -9,8 +11,12 @@ export const SizeStyleBG: Record<'sm' | 'md', string> = {
 };
 
 export const ButtonStyle = 'rounded';
-export const ButtonStyleBG = 'rounded-full transition-colors ' + 'hover:bg-bg-primary-hover active:bg-bg-primary-hover';
+export const ButtonStyleBG = cnTw(
```

Comment:
What do you think to rename it to `ButtonStyleWithBG` to better understand that it's not only button background 

`SizeStyleBG` too